orchagent/portsorch.cpp

```diff
@@ -2585,6 +2586,13 @@ void PortsOrch::doLagTask(Consumer &consumer)
                     if (fvValue(i) == "down")
                     {
                         gNeighOrch->ifChangeInformNextHop(alias, false);
+                        Port lag;
+                        if (getPort(alias, lag))
```

Comment:
Could we please log a message if the getPort(..) function returns false

Reply:
In case the LAG is not there yet, we should not print, in case some error case, it will be printed out later.